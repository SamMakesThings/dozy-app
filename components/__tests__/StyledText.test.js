import React from 'react';
import { shallow } from 'enzyme';
import MonoText from '../StyledText';

describe('StyledText', () => {
  it('should match the snapshot', () => {
    const component = shallow(<MonoText>Snapshot test!</MonoText>);
    expect(component).toMatchSnapshot();
  });
});
