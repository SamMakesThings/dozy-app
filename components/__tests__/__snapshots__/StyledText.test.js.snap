// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledText should match the snapshot 1`] = `
<Text
  style={
    Array [
      undefined,
      Object {
        "fontFamily": "space-mono",
      },
    ]
  }
>
  Snapshot test!
</Text>
`;
